src/Project/Sugcon2024/Sugcon/src/components/Events/Sessions.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { Field } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Field, withDatasourceCheck } from '@sitecore-jss/sitecore-jss-nextjs';
 import useSWR from 'swr';
 import clsx from 'clsx';
 import { Flex } from '@chakra-ui/react';
+import { ComponentProps } from 'lib/component-props';
 
 interface Fields {
   SessionizeUrl: Field<string>;
 }
 
-type SessionsProps = {
-  params: { [key: string]: string };
+type SessionsProps = ComponentProps & {
```

Comment:
Adding the ComponentProps here, introduces errors in the related StoryBook assets, can these be updated as well?

Reply:
Updated.